### PR TITLE
feat: Suppose to be a static site

### PR DESCRIPTION
**Pull request by miniwangdali**
1. Use 404 to redirect back to app page
2. Generate routes before runtime.

Contributes to #5

### CODE REVIEW

File: lacerta-blog/src/404.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<html>
+
+<head>
+  <!-- This stores the URL the user was attempting to go to in sessionStorage,
+    and then redirects all 404 responses to the app’s index.html page -->
+  <!-- See http://www.backalleycoder.com/2016/05/13/sghpa-the-single-page-app-hack-for-github-pages/ -->
+  <script>
+    sessionStorage.redirect = location.href;
+  </script>
+  <meta http-equiv="refresh" content="0;URL='https://miniwangdali.github.io/Lacerta'">
```
Comment (Jozdortraz) [inline lacerta-blog/src/404.html]: Another Host URL need to update

File: lacerta-blog/src/app/app.component.ts [hunk 2]
```diff
@@ -23,18 +23,18 @@ export class AppComponent implements OnInit {
...
+    //     let permalink = p.permalink;
+    //     permalink = permalink.startsWith('/') ? permalink.substring(1) : permalink;
+    //     permalink = permalink.endsWith('/') ? permalink.substr(0, permalink.length - 1) : permalink;
+    //     return {
+    //       path: permalink,
+    //       component: PageComponent,
+    //       pathMatch: 'full'
+    //     };
+    //   })
+    // );
```
Comment (Jozdortraz) [inline lacerta-blog/src/app/app.component.ts]: if something is not used right now, you can simply delete it rather than leaving it as a comment.

**Comment by miniwangdali** [comment]
PR updated. Please feel free to continue the review.

**Comment by Jozdortraz** [comment]
Fabulous!

**Pull request merged**